content/ember-released-5-5.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Ember 5.5 Released
+authors:
+  - jared-galanis
+date: 2023-12-15T00:00:00.000Z
+tags:
+  - releases
+  - '2023'
+  - version-5-x
+---
+
+Today the Ember project is releasing version 5.5 of Ember.js and Ember CLI.
+
+This release kicks off the 5.6 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](https://blog.emberjs.com/new-ember-release-process/)
+- [The Ember Project](https://blog.emberjs.com/ember-project-at-2-0/)
+- [Ember LTS Releases](https://blog.emberjs.com/announcing-embers-first-lts/)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 5.5
+
+Ember.js 5.5 is an incremental, backwards compatible release of Ember with bug fixes, performance improvements, and minor deprecations.
+
+#### Bug fixes
+
+Ember.js 5.5 introduced 2 bug fixes.
+
+- [#20575](https://github.com/emberjs/ember.js/pull/20575) Update backburner.js to 2.4.2 to ensure scheduleOnce works correctly following a cancelled job
+- [#20578](https://github.com/emberjs/ember.js/pull/20578) Deprecate outlet TemplateFactory support
+
+For more details on changes in Ember.js 5.5, please review the [Ember.js 5.5.0 release page](https://github.com/emberjs/ember.js/releases/tag/v5.5.0).
+
+---
+
+## EmberData
+
+EmberData is the official data persistence library for Ember.js applications.
+
+This release cycle EmberData is again electing to not release a new minor.
+
+Currently work in the library is focused primarily on continuing to align 4.12 and 5.3 to enable as many folks in the community to seamlessly upgrade as possible.
```

Comment:
I believe 5.3 is an LTS now for ember-data

Reply:
confirm